### DISCUSSION TITLE
There should be a guide on adding support for new functions and operators in shapes' WHERE clauses

### DISCUSSION THREAD

**Discussion by alco** [electric-sql/electric#1305] [General]
The Electric team can only do so much with our limited resources. However, since Electric is an open-source project, I think many people would love to contribute code to be able to use more data types and/or functions in WHERE clauses of their shape subscriptions. By taking such contributions, we can greatly increase the rate of addition of Postgres functionality to shapes.

**Comment by KyleAMathews** [comment]
👋 we've been working the last month on a rebuild of the Electric server over at a temporary repo https://github.com/electric-sql/electric-next/

You can read more about why we made the decision at https://next.electric-sql.com/about

We're really excited about all the new possibilities the new server brings and we hope you'll check it out soon and give us your feedback.

We're now moving the temporary repo back here. As part of that migration we're closing all the old issues, PRs, and discussions. We really appreciate you taking the time to start this discussion! If it's still relevant in the new system, please reopen.
